tests/unit_tests/test_mcda_without_robustness.py

```diff
@@ -0,0 +1,244 @@
+import unittest
+from unittest import TestCase
+
+from mcda.mcda_without_robustness import MCDAWithoutRobustness
+from mcda.configuration.config import Config
+from mcda.utils import *
+from mcda.utils_for_parallelization import *
+from mcda.utility_functions.aggregation import Aggregation
+
+class TestMCDA_without_robustness(unittest.TestCase):
+
+    @staticmethod
+    def get_test_config():
+        return {
+            "input_matrix_path": "/path/to/input_matrix.csv",
```

Comment:
Is this ` "input_matrix_path": "/path/to/input_matrix.csv",` something that the user needs to update before running these tests?
If so then probably here as well will it be possible to use relative path so that user won't have to update it at several places(in below three methods)?

Reply:
This is only a placeholder and tests run without updating the path. The point is that the` input_matrix_path` is never used within the tests and all needed test matrices are given through functions like `get_input_matrix`.